Projects/Foundations/Schemas/Docs-Tutorials/I/Intro/Intro-to-Trading-Systems/intro-to-trading-systems-003-tutorial-topic-frame-it-flexibly.json

```diff
@@ -20,9 +20,9 @@
                 "updated": 1639147285749
             },
             {
-                "language": "TR",
-                "text": "Özet: Bir Ticaret Sistemi ( Trading System ) , ticaret stratejilerini yönetir ve bunları oluşturmak için esnek, ancak yapılandırılmış bir çerçeve sağlar.",
-                "updated": 1640692400531
+                "language": "RU",
+                "text": "Торговая система регулирует торговые стратегии и обеспечивает гибкую, но структурированную основу для их построения.",
+                "updated": 1640878761889
```

Comment:
Same problem here in this File. When you fix it remember that it isn`t only one paragraph which is affected.